test/kubernetes_unittest.cc

```diff
@@ -23,10 +23,95 @@
 #include "gtest/gtest.h"
 #include "temp_file.h"
 
+#include <boost/network/protocol/http/server.hpp>
 #include <memory>
 
 namespace google {
 
+namespace {
+
+json::value NodeMetadataWithAssociations(const json::value& node) {
+  return json::object({
+    {"blobs", json::object({
+      {"api", json::object({
+        {"version", json::string("1.6")},  // Hard-coded in kubernetes.cc.
+        {"raw", node->Clone()},
```

Comment:
I suggest passing in an owned pointer instead and moving it in here. You can use `Clone()` to create an owned pointer in the caller (so, I'm basically suggesting:
```c++
json::value NodeMetadataWithAssociations(json::value node) {
  ...
      {"raw", std::move(node)},
  ...
}
...
  EXPECT_EQ(NodeMetadataWithAssociations(node->Clone())->ToString(),
            m.metadata().metadata->ToString());
```
Ditto in `PodMetadataWithAssociations` and `ContainerMetadataWithAssociations`.

Reply:
Done.